oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/VocabularyForm.jsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { DetailPageEditForm } from "./DetailPageEditForm";
+import { translateObjectToArray, clearObjectValues } from "../utils";
+import _ from "lodash";
+import { useFormConfig } from "@js/oarepo_ui/forms";
+
+const options = {
+  languages: [
+    { text: "cs", value: "cs" },
+    { text: "en", value: "en" },
+    { text: "de", value: "de" },
+  ],
+};
+
+const VocabularyForm = () => {
+  const { record, formConfig, recordPermissions } = useFormConfig();
+  console.log(record, formConfig, recordPermissions);
+  const { vocabularyProps } = formConfig;
+  const editMode = _.has(formConfig, "updateUrl");
+  const hasPropFields = !_.isEmpty(vocabularyProps);
+  const apiCallUrl = editMode ? formConfig.updateUrl : formConfig.createUrl;
+  const availablePropFields = clearObjectValues(vocabularyProps.props);
+
+  const editModeAndProps = editMode
+    ? { ...availablePropFields, ...record.props }
+    : availablePropFields;
+
+  const propFieldsWithValues = hasPropFields ? editModeAndProps : {};
+
+  const initialValues = editMode
```

Comment:
Regarding the state in formik of multi input, fields. If we check the invenio’s RDM app, we can see that they are also shipping data in the format that is suitable for FieldArray i.e. array for example a specific case 
![image](https://github.com/oarepo/oarepo-vocabularies/assets/34883098/97362ed0-1537-4687-a670-f5b68c4239be)


Also when returning the data from the server in their app, it comes in the shape of Array suitable for formiks’ arrayfield, so probably they are converting this on the BE or maybe they are storing it differently. 

In invenio RDM, when you are going to for example edit a deposit, it receives everything from the backend in form of config, and then it initializes the formik state with this record (maybe make some modifications to the record before passing it to initialize formik’s state). I don’t see how we can let the component itself handle its initial state, because for example, fieldArray component gets its initial data from formik by using formik’s getIn function     const existingValues = getIn(values, fieldPath, []);

In our case, we are not getting some of the fields, such as multilingual field for example, in correct shape, therefore it is not possible to just pass the record to initialize formik’s state. 

It is similar with creating/editing. In context of this app, we don’t have an input that is managing item’s potential parent, meaning item’s parent is not part of formik’s values. This means, if I am for example editing an item, that is not top level item, and if I was just to pass formik’s state without adding anything to it, it would change this item to a top level item, because the parent was not sent in the request.

Reply:
From yesterday's discussion, we will handle this by input component using computed field (`_title`) by its children. `_title` will hold internal representation and `title` will correspond to initial value format. On submit event handler, every prefixed by `_` will be ignored.